docs/knowledge_base/create_project.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Phylum Project Creation
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+# Overview
+
+Phylum projects are used to represent a software project; often a git repository. A Phylum project organizes the dependencies and findings for your software project. This page enumerates the ways a Phylum project can be created.
+
+## Walkthrough
```

Comment:
```suggestion
## Walkthrough

Note: Project creation does not trigger an analysis of the project. Instead, the [`phylum analyze`](https://docs.phylum.io/docs/phylum_analyze) command will trigger an analysis job and organize the results in the specified Phylum project.
```

Do we need to be clear that creating the project will not actually trigger an analysis job?

Reply:
Sounds like a good idea to me...added.